macros/math/PGnumericalmacros.pl

```diff
@@ -17,7 +17,13 @@ =head1 NAME
 
 Numerical methods for the PG language
 
-=head2 Interpolation  methods
+=cut
+
+BEGIN { strict->import; }
+
+sub _PGnumericalmacros_init { }
+
+=head2 Interpolation methods
 
 =head3 Plotting a list of points (piecewise linear interpolation)
```

Comment:
This header is to long, and it would be better if it were just the method name.  The current header could be a remark below.  So make this something like
```suggestion
=head3 plot_list

Plot a list of points (piecewise linear interpolation).
```
So throughout this file, all of the `=head3` lines should just be the method name.  Then when a problem author is trying to find a method and what it does, it will be listed in the sidebar in the generated POD.  In general this is the scheme that I would like to work toward in the POD for all of the macro files.